core/storage/skill/gae_models.py

```diff
@@ -205,7 +205,8 @@ def get_unpublished_by_creator_id(cls, user_id):
         """
         return cls.query(
             cls.creator_id == user_id,
-            cls.skill_is_private == True) #pylint: disable=singleton-comparison
+            cls.skill_is_private == True, #pylint: disable=singleton-comparison
```

Comment:
nit, here and below: add a space after `#`

Reply:
@tjiang11 -- flagging this.

Reply:
done